Gemfile

```
@@ -29,6 +29,7 @@ gem "docraptor", "~> 1.3.0"
 gem "email_address", "~> 0.1.11"
 gem "full-name-splitter", "~> 0.1.2"
 gem "gibbon", "~> 3.3.0"
+gem "heroku-deflater", "~> 0.6.3", group: "production"
```

should we not do something like:

```suggestion
group :production do
  gem "heroku-deflater", "~> 0.6.3"
end
```

---

We'll switch to that format as soon as we have more than 1 production env only dep.